plugins/module_utils/network/sonic/config/logging/logging.py

```diff
@@ -287,6 +287,10 @@ def _state_overridden(self, want, have):
         commands = []
         requests = []
 
+        if (have['remote_servers'] == want['remote_servers']) and (have['security_profile'] == ""):
+            if 'security_profile' not in want:
+                want['security_profile'] = ""
+
```

Comment:
have would not contain empty 'security_profile', hence this check can be removed.
```suggestion
```